samples/3.1/MvcSample/DoNothingCorrelationIdProvider.cs

```diff
@@ -1,13 +1,11 @@
+using System.Threading.Tasks;
 using CorrelationId.Abstractions;
 using Microsoft.AspNetCore.Http;
 
 namespace MvcSample
 {
     public class DoNothingCorrelationIdProvider : ICorrelationIdProvider
     {
-        public string GenerateCorrelationId(HttpContext context)
-        {
-            return null;
-        }
+        public Task<string> GenerateCorrelationId(HttpContext _) => null;
```

Comment:
Should this implementation be `Task.FromResult<string>(null);` instead?

Reply:
Good point.